packages/legend-query/src/QueryBuilder_Const.ts

```diff
@@ -57,6 +57,20 @@ export enum SUPPORTED_FUNCTIONS {
   LESS_THAN = 'meta::pure::functions::lang::tests::lessThan',
   LESS_THAN_EQUAL = 'meta::pure::functions::lang::tests::lessThanEqual',
   STARTS_WITH = 'meta::pure::functions::string::startsWith',
+  ADJUST = 'meta::pure::functions::date::adjust',
```

Comment:
let's add a newline here to separate these and the operations, also please remove the empty line `56`